doc/io_streams.rdoc

```diff
@@ -193,96 +124,80 @@ A stream is automatically closed when claimed by the garbage collector.
 
 Attempted reading or writing on a closed stream raises an exception.
 
+The relevant methods:
+
 - IO#close: Closes the stream for both reading and writing.
 - IO#close_read: Closes the stream for reading; not in ARGF.
 - IO#close_write: Closes the stream for writing; not in ARGF.
 - IO#closed?: Returns whether the stream is closed.
 
 ==== End-of-Stream
 
-You can query whether a stream is positioned at its end using
-method IO#eof? (also aliased as +#eof+).
+You can query whether a stream is positioned at its end:
 
-You can reposition to end-of-stream by reading all stream content:
+- IO#eof? (also aliased as +#eof+): Returns whether the stream is at end-of-stream.
+
+You can reposition to end-of-stream by reading all stream content
+or by using method IO#seek:
 
   f = File.new('t.txt')
   f.eof? # => false
   f.read # => "First line\nSecond line\n\nFourth line\nFifth line\n"
   f.eof? # => true
 
-Or by using method IO#seek:
```

Comment:
I prefer this old format that separates the two examples. I find longer examples to be hard to read. I also prefer if we showed `IO#seek` first since it avoids reading all the contents of the file (which can be slow if the file is very large).

Reply:
Done.